airflow/providers/apprise/hooks/apprise.py

```
@@ -0,0 +1,127 @@
+            apprise_obj.add(config_object["path"], tag=config_object.get("tag", None))
+        else:
+            raise ValueError("only dict / list[dict] type of values are expected")
+
+    def notify(
+        self,
+        body: str,
+        title: str | None = None,
+        notify_type: NotifyType = NotifyType.INFO,
+        body_format: NotifyFormat = NotifyFormat.TEXT,
+        tag: str | Iterable[str] | None = None,
+        attach: str | None = None,
+        interpret_escapes: bool | None = None,
+        config: AppriseConfig | None = None,
+    ):
+        r"""
+        Send message to plugged-in services
+
+        :param body: Specify the message body
+        :param title: Specify the message title. This field is complete optional
+        :param notify_type: Specify the message type (default=info). Possible values are "info",
+            "success", "failure", and "warning"
+        :param body_format: Specify the input message format (default=text). Possible values are "text",
+            "html", and "markdown".
+        :param tag: Specify one or more tags to filter which services to notify
+        :param attach: Specify one or more file attachment locations
+        :param interpret_escapes: Enable interpretation of backslash escapes. For example, this would convert
+            sequences such as \n and \r to their respected ascii new-line and carriage
+        :param config: Specify one or more configuration
+        """
+        title = title or ""
+
+        apprise_obj = apprise.Apprise()
+        if config:
+            apprise_obj.add(config)
+        else:
+            self.set_config_from_conn(apprise_obj)
+        apprise_obj.notify(
+            body=body,
+            title=title,
+            notify_type=notify_type,
+            body_format=body_format,
+            tag=tag,
+            attach=attach,
+            interpret_escapes=interpret_escapes,
+        )
+
+    def get_conn(self) -> None:
+        pass
```

I wonder if we should raise here instead?

---

Yup, I think it's always better to fail loudly.